packages/react-apps-toolkit/README.md

```diff
@@ -0,0 +1,24 @@
+# React Toolkit for Contentful Apps
+
+This package consists of helpers and hooks to create a Contentful app with React. 
+
+It is an extension of the official React launcher from the [`create-contentful-app`](https://github.com/contentful/create-contentful-app/tree/master/packages/create-contentful-app) package,
+but can also be used with any other React app bootstrapped with Contentfuls [app-sdk](https://github.com/contentful/ui-extensions-sdk).
+
+
+## Installation
+
+```shell
+npm install --save @contentful/react-apps-toolkit
```

Comment:
Things are automatically saved for a few years now

```suggestion
npm install @contentful/react-apps-toolkit
```

Reply:
Then we should also update [this](https://github.com/contentful/node-apps-toolkit#installation)

Reply:
This is just a not. Not broken, but unnecessary additional characters.